vdb-bench-assembly/plugins/vdb-bench-dataservice/dataservicePageController.js

```diff
@@ -133,7 +133,7 @@
             return ConnectionSelectionService.getConnectionState(conn);
         };
 
-        vm.selectPage(DSPageService.DS_HOME_PAGE);
+        vm.selectPage(DSPageService.DATASERVICE_SUMMARY_PAGE);
     }
```

Comment:
This appears odd to me. We have a ds-home-page and never use it? There seems little purpose for the home page other than it being the introductory landing page for loading on startup. Are these changes not actually converting the summary page into the home page?